src/EncryptedData.js

```diff
@@ -438,7 +438,7 @@ export default class EncryptedData {
 		
 		let hmacHashAlgorithm = "SHA-1";
 		
-		if("prf" in pbkdf2Params)
+		if("prf" in pbkdf2Params && pbkdf2Params.prf.algorithmId)

```

Comment:
@pavel-kubik It is nearly impossible to have empty string as "algorithmId". It could be the case but only if you have incorrectly formed initial data. So, do not think PKI.js should handle such cases.